flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -107,6 +107,10 @@ fir::ExtendedValue Fortran::lower::getAbsentIntrinsicArgument() {
 static bool isAbsent(const fir::ExtendedValue &exv) {
   return !fir::getBase(exv);
 }
+static bool isAbsent(llvm::ArrayRef<fir::ExtendedValue> args,
+                     unsigned argIndex) {
```

Comment:
I think `llvm::ArrayRef::size` returns a `size_t`. [https://llvm.org/doxygen/classllvm_1_1ArrayRef.html#a85ffb6531d4cda988ea81f18d4e56fb7] Conversions to `unsigned` could possibly give a warning under some platform/compiler/option settings. (Hopefully, not.)